kubernetes/loculus/values.schema.json

```diff
@@ -709,12 +709,13 @@
         "tag": {
           "groups": ["image"],
           "type": "string",
-          "description": "The tag to pull. Examples: latest, 0.5.7"
+          "description": "The tag to pull. Examples: latest, 0.5.7. Can include a digest to absolutely pin the image (tags can move): v0.7.10.1@sha256:96e431192d6493ed71ac10bc273710df21e7c520a77bebaa7d94ff859264d36c"
         },
         "pullPolicy": {
           "groups": ["image"],
           "type": "string",
-          "description": "The pull policy to use. Examples: IfNotPresent, Always."
+          "enum": ["Always", "IfNotPresent", "Never"],
+          "description": "The pull policy to use. Always: always pull the image. IfNotPresent: pull if not present locally. Never: never pull, use local image only. If not specified, defaults to Always if tag is 'latest' or empty, otherwise IfNotPresent."
```

Comment:
this is a bit unclear: `The default value is IfNotPresent, setting the tag to 'latest' will default to Always`.

It is probably not possible to leave the field empty (`pullPolicy: `)?

Reply:
hmm the schema requires it at the moment it seems
<img width="748" height="202" alt="image" src="https://github.com/user-attachments/assets/cb561489-3a93-4334-9912-8248342d196d" />

Reply:
just to confirm there is no default now?